SciPy2024/module1/llms-and-prompt-engineering-with-olmo.ipynb

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ssec_tutorials.scipy_conf.parse_olmo_chat_completion_response(chat_response)
</pre><p>You'd probably wanna just import this function directly above instead to be shorter:</p>

<pre class="ql-syntax" spellcheck="false">from ssec_tutorials.scipy_conf import parse_olmo_chat_completion_response

parse_olmo_chat_completion_response(chat_response)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-ssec/tutorials/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>